locales/zhCN.lua

```diff
@@ -1175,22 +1175,22 @@ for key, value in pairs({
 	[181638] = "通缉布告",	-- Wanted Poster
 	[181672] = "稻草人",	-- Wickerman Effigy	--TODO: This was taken from tbc Wowhead
 	--TODO: [181698] = "Voidstone",	-- Voidstone
-	--TODO: [181748] = "Blood Crystal",	-- Blood Crystal
+	[181748] = "血水晶",	-- Blood Crystal
```

Comment:
Where did you get this object name (and others too?) It's not on wowhead

Reply:
I have discovered the bug with localization thanks to this, I was looking up retail->classic->tbc for newly added objects but forgot to do the same for old TODOs, thank you.